tv/lib/dl_daemon/download.py

```
@@ -530,16 +532,16 @@ def pick_initial_filename(self, suffix=".part", torrent=False,
         if not torrent:
             # this is an ascii filename and needs to be fixed
             filename = clean_filename(filename)
-
-        full_path = os.path.join(download_dir, filename)
-        if is_directory:
-            # if this is a torrent and it's a directory of files, then
-            # we create a temp directory to put the directory of files
-            # in.
-            new_filename = next_free_directory(full_path)
-        else:
-            new_filename, fp = next_free_filename(full_path)
-            fp.close()
+        new_filename = os.path.join(download_dir, filename)
+        if not exists:
+            if is_directory:
+                # if this is a torrent and it's a directory of files, then
+                # we create a temp directory to put the directory of files
+                # in.
+                new_filename = next_free_directory(new_filename)
+            else:
+                new_filename, fp = next_free_filename(new_filename)
+                fp.close()
```

Should we throw an error in that case, or just create the path?

UPDATE: decided to log a warning, but create the path